.gitignore

```diff
@@ -0,0 +1,3 @@
+# IDE files

```

Comment:
why are these modifications in the CL still?
it seems to me they were already merged in as part of #241 ?